.github/workflows/codeql-analysis.yml

```diff
@@ -22,6 +22,10 @@ on:
 
 jobs:
   analyze:
+    permissions:
```

Comment:
Thanks for the new update.  Sorry for not noticing this before, but based on [the codeql example](https://github.com/github/codeql-action#usage) we shouldn't need these read permissions either as it's not a private repository.  I think the only thing we need is `security-events:write`

Reply:
agreed. I have made those changes. Thanks!